File: Entmaa/app/src/main/java/com/team/entmaa/data/model/dto/posts/CommentDto.kt
package com.team.entmaa.data.model.dto.posts

import com.team.entmaa.data.model.dto.users.UserDto
import java.io.Serializable

import java.time.ZonedDateTime

class CommentDto : Serializable {
    var id:Int = 0
    var commentedBy:UserDto = UserDto()
    var commentText:String = ""
    var timePosted:ZonedDateTime = ZonedDateTime.now()
}

File: Entmaa Community Service/Entmaa Web Services/DTOs/Posts/CommentDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Entmaa_Web_Services.DTOs.Posts
{
    public class CommentDTO
    {
        public int Id { get; set; }

        public int UserId { get; set; }

        public string CommentText { get; set; }

        public DateTime TimePosted { get; set; }

        public UserDTO CommentedBy { get; set; }
    }
}